app/views/layouts/application.html.haml

```diff
@@ -1,15 +1,30 @@
 !!!
 %html
   %head
-    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %meta{:content => 'text/html; charset=UTF-8', 'http-equiv' => 'Content-Type'}/
     %title Chronos
     = csrf_meta_tags
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
   %body
     = render partial: 'layouts/shared/header'
     = render partial: 'layouts/shared/navbar'
-    /= tabs
+
+    - flash.each do |name, msg|
```

Comment:
Can we actually see what this looks like with some screenshots?

Reply:
Good point.

![screen shot 2018-03-12 at 12 30 40 pm](https://user-images.githubusercontent.com/23364580/37296412-4a9099c6-25f1-11e8-80f4-0214e86de994.png)

For comparison here's the same messages look like on umaps.

![screen shot 2018-03-12 at 12 30 44 pm](https://user-images.githubusercontent.com/23364580/37296428-5bf5d988-25f1-11e8-8ea0-8e2195766622.png)

Reply:
Nice job!